docs/api/materials/ShaderMaterial.html

```diff
@@ -337,8 +337,12 @@ <h3>[property:Boolean isShaderMaterial]</h3>
 			You should not change this, as it used internally for optimisation.
 		</div>
 
+		<h3>[property:Boolean unrollLoops]</h3>
+		<div>
+			Used to optimize shader code at runtime, specifically all loops in the shader code will be expanded. Default is *false*.<br /><br />
 
-
+			You should set this to true, if you want to avoid problems with modern GPUs.
```

Comment:
I think this should be reworded. No-one doesn't want to avoid problems.

Reply:
For backwards compatibility, the default was set to false. If we could have it set to true, this text could be removed.
Alternatively, maybe you have a suggestion on a better wording?